src/genie/libs/parser/iosxe/show_ap.py

```diff
@@ -0,0 +1,1446 @@
+import re
+import json
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# ===========================
+# Schema for:
+#  * 'show ap config general'
+# ===========================
+class ShowApConfigGeneralSchema(MetaParser):
+    """Schema for show ap config general."""
+
+    schema = {
+        "ap_config_general_info": {
+            str: {
+                Optional("cisco_ap_identifier"): str,
```

Comment:
all of them are `Optional`? if there are mandatories one, please remote from them.